Mobile/starter_project_matador/lib/features/login_page/Presentation/widgets/welcome_back.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+import 'constants/global_variables.dart';
+
+class WelcomeBack extends StatelessWidget {
```

Comment:
Since this is just a static widget, you should just call the Text widget and remove this one. The style, if you wanna make it reusable, can be added to the utils folder so that it can be reused everywhere.